MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+
+N> Class cannot be directly set as data type for picker editor in this complex type scenario.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ItemsSourceProvider = new DataFormItemsSourceProvider();
+    dataForm.DataObject = new ContactModel();
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+    dataForm.RegisterEditor("EmployeeDetails", DataFormEditorType.Picker);
+ 
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "EmployeeDetails" && e.DataFormItem is DataFormPickerItem pickerItem)
+        {   
+            pickerItem.DisplayMemberPath = "ID";
+            pickerItem.SelectedValuePath = "Name";
+        }
+    } 
+ 
+    public class DataFormItemsSourceProvider  : IDataFormSourceProvider
+    {
+        public object GetSource(string sourceName)
+        {
+            if (sourceName == "EmployeeDetails")
+            {
+                List<EmployeeInfo> details = new List<EmployeeInfo>();
+                details.Add(new EmployeeInfo() { ID = 1, Name =  "Eric"  });
+                details.Add(new EmployeeInfo() { ID = 2, Name = "James"  });
+                details.Add(new EmployeeInfo() { ID = 3, Name = "Jacob"  });
+                details.Add(new EmployeeInfo() { ID = 4, Name = "Lucas"  });
+                details.Add(new EmployeeInfo() { ID = 5, Name = "Mark"  });
+                details.Add(new EmployeeInfo() { ID = 6, Name = "Aldan"  });
+                details.Add(new EmployeeInfo() { ID = 7, Name = "Aldrin"  });
+                details.Add(new EmployeeInfo() { ID = 8, Name = "Alan"  });
+                details.Add(new EmployeeInfo() { ID = 9, Name = "Aaron"  });
+
+                return details;
+            }
+        return new List<string>();
+        }
+    }
+
+    public class ContactInfo
+    {
+        [Display(Name ="First Name")]
+        public String FirstName { get; set; } 
+        public string EmployeeDetails { get; set; }
+    }
+
+    public class EmployeeInfo
+    {
+        public int ID { get; set; }
+        public string Name { get; set; }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## RadioGroup editor
+
+In the [RadioGroup](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) editor, the [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) control is loaded.
+
+The `items` for [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) is generated for `enum` and `List` data type properties. In order to add [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) editor in the DataForm, you need to register editor as DataFormEditorType.RadioGroup for the required property by using the `RegisterEditor` method.
```

Comment:
The `ItemsSource` for the radio group editor is generated for `enum` data type property. In order to add [RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0) editor in the DataForm, you need to register the editor as `DataFormEditorType.RadioGroup` for the required property by using the `RegisterEditor` method.

Reply:
content updated